account-management-api/tests/account-management-api-stub.test.ts

```diff
@@ -1,4 +1,4 @@
-import { handler, Response } from "../account-management-api";
+import { handler, Response } from "../account-management-api-stub";
```

Comment:
Is there is reason were are appending stub in front of `account-management-apj` . We are in a stub repo so I would assume everything in it is stub .

Reply:
I did this because it was helpful for the artefacts in AWS to have 'stub' in the name so since adding stub to all the infrastructure like the lambda name I was keeping it consistent with the function code. 
Happy to change if that seems unnecessary but i also found it useful when searching for files in my vs code etc